kickstarter/android-oss
# NTV-365  Migrate to Kotlin `DiscoveryAdapter.java`

# 📲 What
Migrate to Kotlin `DiscoveryAdapter.java`

# 🛠 How
**Add KSListAdapter**
1. Create new 4 adapters 
2. use `ConcatAdapter ` to update Discovery Fragment 

![image](https://user-images.githubusercontent.com/1075310/151856656-2e8ee761-7eab-41e9-9690-3bac248137d4.png)

# 👀 See

https://user-images.githubusercontent.com/1075310/151859927-eb758406-1589-42a5-8f2c-37895ae230e5.mp4


# 📋 QA
Change tabs on Discovery and make sure everything works

- open the app logged in
- open the app without being logged

# Story 📖
https://kickstarter.atlassian.net/browse/NTV-365
